apps/assessment_registry/management/commands/migrate_old_assessments.py

```diff
@@ -332,6 +332,8 @@ def _get_bg_crisis_type():
 
         }
         assessment_reg, created = AssessmentRegistry.objects.get_or_create(**input_data)
+        assessment_reg.created_at = old_ary.created_at
+        assessment_reg.save()
```

Comment:
Let's use `.update` instead of this. We still need to use the same data for updated_at as well.

Maybe we can copy all created/updated information from existing ARY to new ones as a last step in the migration script.
```python
AssessmentRegistry.objects.update(
    created_at=models.F('lead__assessment__created_at'),
    created_by=models.F('lead__assessment__created_by'),
    updated_at=models.F('lead__assessment__updated_at'),
    updated_by=models.F('lead__assessment__updated_by'),
)
```